library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -599,6 +603,8 @@ public SimpleTooltip build() throws IllegalArgumentException {
                 SimpleTooltipUtils.setTextAppearance(tv, mDefaultTextAppearanceRes);
                 tv.setBackgroundColor(backgroundColor);
                 tv.setTextColor(textColor);
+                tv.setTypeface(Typeface.createFromAsset(context.getAssets(), fontPath));
+                tv.setText(fontSize);
```

Comment:
I think `fontSize` shouldn't be set to `setText`.

Reply:
hi @douglasjunior  , thank you ;
 oh ,sorry 
I did'nt test before PR.

I will edit these bugs and commit  again